Base/usr/share/man/man2/geteuid.md

```diff
@@ -0,0 +1,25 @@
+## Name
+
+geteuid, getegid - get effective user / group id
+
+## Synopsis
+
+```**c++
+#include <unistd.h>
+
+uid_t geteuid(void);
+gid_t getegid(void);
+```
+
+## Description
+
+Returns the effective user or group id.
+
+## See also
+
+* [`setuid_overview`(7)](../man7/setuid_overview.md)
+* [`getuid`(2) / `getgid`(2)](getuid.md)
+* [`getresuid`(2) / `getresgid`(2)](getresuid.md)
+* [`seteuid`(2) / `setegid`(2)](seteuid.md)
+* [`setuid`(2) / `setgid`(2)](setuid.md)
+* [`setresuid`(2) / `setresgid`(2)](setresuid.md)
```

Comment:
Can we perhaps not have this many separate pages, and put everything into one or two pages instead?

Reply:
How'd you organize it? Getters on one page, setters on another?

Reply:
Honestly I'd just stuff everything here into one page 🙂

Reply:
I think I'd like to start with a bunch of pages. I kind of like it as-is at the moment. It's easy to consolidate later if we want to.